src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -22,6 +25,25 @@ const Body = SelectAuthenticatorVerifyViewBody.extend({
       }
     }];
   },
+
+  // If the `deviceKnown` attribute is true, we should put the signed_nonce method to the top of authenticator list.
+  // This is in sync with v2/ion/ui-schema/ion-object-handler.js - createOVOptions
+  _sortMethodOptionsIfNeeded(methodOptions) {
+    // Check if the `deviceKnown` attribute is true
+    const deviceKnown = this.options.currentViewState.relatesTo?.value?.deviceKnown;
```

Comment:
can we also check if options and currentViewState are not undefined? Just so we are covered
```
this.options?.currentViewState?.relatesTo?.value?.deviceKnown;
```

Reply:
I updated the code